root/etc/e-smith/events/actions/nethserver-blacklist-conf

```diff
@@ -27,15 +27,30 @@ event=$1    # name of the event (not used)
 type=$2     # blacklist type (ipsets or dnss)
 
 if [[ "$type" == "ipsets" ]]; then
-    conf="blacklist"
+    status=$(/sbin/e-smith/config getprop blacklist status)
+    ipsetUrl=$(/sbin/e-smith/config getprop blacklist Url)
+    geoipStatus=$(/sbin/e-smith/config getprop blacklist geoipStatus)
+
+    if [[ "$status" != "enabled" ]]; then
+        exit 0
+    fi
+
+    if [[ "$ipsetUrl" != "" ]]; then
+        /usr/share/nethserver-blacklist/download ipsets
+    fi
```

Comment:
Is this a real condition?

Reply:
See the comment below, the URL is not more a mandatory, we use it like a new checkbox in the UI, if the URL is a valid git repository and if the URL exists then we download the list and use this list to the cronjob

Reply:
Not sure if it's a good choice, I need to test it.
I will do it probably tomorrow :)